lib/ansible/modules/monitoring/zabbix/zabbix_host.py

```diff
@@ -234,6 +234,47 @@
         type: bool
         default: 'yes'
         version_added: '2.0'
+    macros:
+        description:
+            - List of user macros to assign to the zabbix host.
+            - Providing I(macros=[]) with I(force=yes) will clean all of the existing user macros from the host.
+        type: list
+        elements: dict
+        version_added: '2.10'
+        suboptions:
```

Comment:
```python
            description:
                description:
                    - Description of the user macro.
                type: str
                required: false
```

Reply:
Seems to be supported only with 4.4 and above. Done, thanks for noticing